CODE_OF_CONDUCT.md

```diff
@@ -33,7 +33,7 @@ Examples of unacceptable behavior include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or email
   address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
+* Other conduct that could reasonably be considered inappropriate in a
```

Comment:
Likewise, this was correct?

Reply:
but grammitically it isn't correct if you see

Reply:
ChatGPT says they are both grammatically correct and based on writer's preference or the intended audience (American vs. British English)